Libraries/Components/TabBarIOS/TabBarIOS.ios.js

```diff
@@ -27,6 +27,9 @@ class TabBarIOS extends React.Component {
     barTintColor?: $FlowFixMe,
     translucent?: boolean,
     itemPositioning?: 'fill' | 'center' | 'auto',
+    onTVFocus?: $FlowFixMe,
```

Comment:
Hmm, having a bunch of TV-specific props is pretty ugly. Can these just be onFocus etc and document that they're Apple TV only? Can you also document a bit more what these events do?